src/demo-app/data-table/data-table-demo.scss

```diff
@@ -0,0 +1,86 @@
+/* Structure */
+.demo-table-container {
+  display: flex;
+  flex-direction: column;
+  max-height: 800px;
+
+  // Table fills in the remaining area with a scroll
+  .cdk-table {
+    flex: 1 1 auto;
+    overflow: auto;
+  }
+}
+
+/* Material Styles */
```

Comment:
Changed the comment to the following. Let me know if its not totally clear

```

/*
 * Styles so that the demo's cdk-table matches the material design spec
 * https://material.io/guidelines/components/data-tables.html
 */
```